play/playstore-description.yml

```diff
@@ -58,6 +56,8 @@ feature_11: >
 unavailable_documents: ESV, NIV and NKJV
 feature_12: >
   Offline: the only time you need internet is for downloading documents.
+feature_13: >
+  Study Pads for easy note-writing and Bible reference adding during listening to sermons.
```

Comment:
Could change "during" to "while" to prevent all the "ing" sounds in the sentence. And would improve grammar a bit too.

Reply:
Fixed